### ISSUE TITLE
Persona: Icons should have better color contrast.

### ISSUE THREAD

**Issue by aneeshack4** [OfficeDev/office-ui-fabric-react#10854]
### Environment Information

- **Package version(s)**: (fill this out)
- **Browser and OS versions**: (fill this out)

> Note that our screen reader support is limited to Edge + Narrator. Issues that do not reproduce in Edge + Narrator, and aren't caused by obvious invalid aria values, should be filed with the respective screen reading software, not the Fabric repo.

### Describe the issue:
Bug 4 of #7643: All the initial icons of 'Persona' don't have sufficient color contrast.

### Please provide a reproduction of the issue in a codepen:
![image](https://user-images.githubusercontent.com/4161791/66877763-cd43b980-ef84-11e9-880a-a964747e53d2.png)

#### Actual behavior:

'Color contrast for the initial icons of persona under 'Persona with initials' variant is 3.2:1 which is not sufficient 

#### Expected behavior:

'All the Elements must have the color contrast ration greater than or equal to 4.5:1.

**Comment by micahgodbolt** [comment]
persona colors were revamped in 7.0 to meet contrast ratios. The light green and pink are both 4+ now.
